app/frontend/src/App.tsx

```diff
@@ -17,7 +17,7 @@ import { theme } from "./theme";
 
 const useStyles = makeStyles((theme) => ({
   padding: {
-    paddingBottom: theme.spacing(7),
+    paddingBottom: theme.spacing(10),
```

Comment:
How did you choose 10? This will conflict with my PR awaiting review and I did 9 (can't remember why) but I could change it to 10.

Reply:
I chose 10 because in figma there is 16px gap between navigation and button. Navigation has 64px (Doesn't matter what device I check on) so if I put 10 padding bottom evaluates to 80px (64+16). But what I did not see is that in Figma navigation is slightly bigger (73px). So it's up to you ;)